cmake/serac-config.cmake.in

```diff
@@ -117,8 +117,12 @@ if(NOT SERAC_FOUND)
       set(CALIPER_DIR ${SERAC_CALIPER_DIR}) 
     endif()
     # See comment in SetupSeracThirdParty.cmake
-    if(SERAC_ENABLE_CUDA AND CMAKE_VERSION VERSION_GREATER_EQUAL 3.17)
-      find_package(CUDAToolkit REQUIRED)
+    if(SERAC_ENABLE_CUDA)
+      if(CMAKE_VERSION VERSION_LESS 3.17)
+        message(FATAL_ERROR "Serac+Caliper+CUDA require CMake > 3.17.")
```

Comment:
gross. Thank you for figuring this out!

Reply:
yea its definitely a niche failure... building Serac (via config-build enforcing CMake version) then building the example w/ the system (non-module loaded cmake version) creates this failure